content-cache/src/nginx_manager.py

```diff
@@ -58,6 +66,25 @@
 # This should be tested with integration tests.
 
 
+@dataclass
+class NginxLuaSection:
+    """Simple class to be used by python-nginx to insert Lua code in Nginx conf.
+
+    Attrs:
+        name: the name of the Lua code section to generate (for instance: "content_by_lua_block")
```

Comment:
I think the team generally use full sentence for the docstrings, i.e., Cap the first letter and add a period at the end.

Reply:
Thanks. I fixed it in cdf94c57a53954759b47a0f851b893964c36bf05.